Code/Tools/ProjectManager/Source/Application.cpp

```diff
@@ -24,6 +27,73 @@
 #include <QInputDialog>
 #include <QIcon>
 
+#if defined(AZ_PLATFORM_WINDOWS)
+LONG WINAPI CreateMiniDump(struct _EXCEPTION_POINTERS* exceptionPointers)
+{
+    AZ::IO::FixedMaxPath dumpPath{ "@log@/O3DEDump.dmp" };
+    if (auto fileIoBase = AZ::IO::FileIOBase::GetInstance(); fileIoBase != nullptr)
+    {
+        dumpPath = fileIoBase->ResolvePath(dumpPath, "@log@/O3DEDump.dmp");
+    }
+
+    // FixedMaxPath appends '\x1' to the end of the path string which causes CreateFileW to fail, remove it
+    AZStd::string dumpPathStr = dumpPath.String();
+    dumpPathStr = dumpPathStr.substr(0, dumpPathStr.length() - 1);
+    const char* dumpPathRaw = dumpPathStr.c_str();
```

Comment:
Can we use a `string_view` here instead?

```suggestion
    const AZStd::string_view dumpPathStr { dumpPath };  // might need to be dumpPath.c_str() or other
```

and later
```
    const AZStd::to_wstring(dumpPathWStr, dumpPathStr.substr(0, dumpPathStr.length() - 1) );
```

Reply:
I wanted to use string_view but it made printing the errors more awkward. Since this code is only called once while cool it is kind of a micro-optimization.

Reply:
Ah, yeah I was hoping string_view would make the code more readable, but if it makes printing errors more unreadable then it isn't helping.